extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -106,15 +107,23 @@ async function handleObjectPropertiesDialogCommand(context: azdata.ObjectExplore
 		return;
 	}
 	try {
-		const parentUrn = await getParentUrn(context);
+		const serverInfo = await azdata.connection.getServerInfo(connectionUri);
+		if (serverInfo) {
```

Comment:
Did you mean to add this?

Reply:
let me remove it. I was using it for testing.